spec/models/ems_refresh/refreshers/azure_refresher_spec.rb

```diff
@@ -78,6 +78,8 @@ def assert_specific_flavor
       :supports_hvm             => nil,
       :supports_paravirtual     => nil,
       :block_storage_based_only => nil,
+      :root_disk_size           => 1_072_693_248,
+      :swap_disk_size           => 71_680
```

Comment:
The test might be clearer if written out with Rails numbers (e.g. `70.kilobytes`)